app/controllers/service_providers_controller.rb

```diff
@@ -47,7 +58,16 @@ def show; end
 
   def all
     return unless current_user.admin?
-    @service_providers = ServiceProvider.all.sort_by(&:created_at).reverse
+    prod_apps = ServiceProvider.all.sort_by(&:created_at).reverse.select { 
+                  |sp| sp.prod_config == false 
+                }
+    sandbox_apps = ServiceProvider.all.sort_by(&:created_at).reverse.select { 
+                  |sp| sp.prod_config == false 
+                }
+
+    @service_providers = { 'Production Apps' => prod_apps,
+                           'Sandbox Apps' => sandbox_apps }
+    @show_created_at = true
```

Comment:
rather than defining `@show_created_at` here, does it make sense to define it in the template itself?


ie, in the `app/views/service_providers/all.html.erb`

```
<%= render partial: 'service_provider_table', locals: { service_providers: @service_providers, show_created_at: true } %>
```

and in `app/views/service_providers/index.html.erb`

```
<%= render partial: 'service_provider_table', locals: { service_providers: @service_providers, show_created_at: false } %>
```

Reply:
I can dig that. Changing.